gc/verbose/schema.xsd

```diff
@@ -330,6 +332,9 @@ xmlns:vgc="http://www.ibm.com/j9/verbosegc">
 	</complexType>
 
 	<complexType name="cycle-end">
+		<sequence>
+			<element ref="vgc:heap-fixup" maxOccurs="1" minOccurs="0" />
+		</sequence>
```

Comment:
don't understand why is this related to cycle-end!?

Reply:
Not expert on this, and from a quick look in xsd files we probably have not way of saying this occurs between cycle-start and cycle-end elements. At least we don't sat it for some other similar tags like allocation-stats, heap-resize etc.

However, we do seem to be able to say it may occur withing gc-op scavenge group....

But that's probably not the same thing, since cycle-start/end are single line independent elements (each have their own closing tag with '/'), while 'gc-ops' are multi-line elements, and can control what subelements end up before their closing tags.

Long story short, this is probably not need, possibly harmful, unless I'm missing something else.

Reply:
I see, I will remove this and retest. Could very likely not be needed.

Reply:
Retest passed, marking this conversation as resolved.